tutorials/01_introductory/lifecycle.py

```diff
@@ -0,0 +1,266 @@
+"""
+=======================
+The Lifecycle of a Plot
+=======================
+
+Going from start to finish of one plot in Matplotlib.
```

Comment:
actually you're right, I don't think the sentence is needed, will remove in next push!